### PR TITLE
Add MSFT models to OpenVino ONNX CI check

### PR DESCRIPTION
**Pull request by rblaczkowski**
Add MSFT models to OpenVino ONNX CI check and update xfail tests

### CODE REVIEW

File: ngraph/python/tests/test_onnx/model_zoo_preprocess.sh [hunk 1]
```diff
@@ -36,37 +36,44 @@ done
 shift $((OPTIND -1))
 
 if [ $MODELS_DIR = false ] ; then
-	echo "Unknown location of the ONNX ZOO models"
+	echo "Unknown location of the ZOO models"
 	exit 170
 fi
+ONNX_MODELS_DIR=$MODELS_DIR/model_zoo/onnx_model_zoo
+MSFT_MODELS_DIR=$MODELS_DIR/model_zoo/MSFT
```
Comment (tsocha) [inline ngraph/python/tests/test_onnx/model_zoo_preprocess.sh:43]:
```suggestion
fi

ONNX_MODELS_DIR=$MODELS_DIR/model_zoo/onnx_model_zoo
MSFT_MODELS_DIR=$MODELS_DIR/model_zoo/MSFT
```

File: ngraph/python/tests/test_onnx/model_zoo_preprocess.sh [hunk 2]
```diff
@@ -36,37 +36,44 @@ done
...
 mv output2.pb output_2.pb
 # fix roberta model
-cd $MODELS_DIR/text/machine_comprehension/roberta/model/roberta-sequence-classification-9/
+cd $ONNX_MODELS_DIR/text/machine_comprehension/roberta/model/roberta-sequence-classification-9/roberta-sequence-classification-9
 mkdir test_data_set_0
 mv *.pb test_data_set_0/
+
+# Prepare MSFT models
+wget -O $MSFT_MODELS_DIR/20191107.zip https://onnxruntimetestdata.blob.core.windows.net/models/20191107.zip
+unzip $MSFT_MODELS_DIR/20191107.zip -d $MSFT_MODELS_DIR/20191107 && rm $MSFT_MODELS_DIR/20191107.zip
```
Comment (tsocha) [inline ngraph/python/tests/test_onnx/model_zoo_preprocess.sh:79]: this file is pretty big so maybe its worth to add an option to skip(or enable) download of it?

Comment (rblaczkowski) [reply ngraph/python/tests/test_onnx/model_zoo_preprocess.sh]: by default it was set as disabled - to download this special flag should be used

File: ngraph/python/tests/test_onnx/model_zoo_preprocess.sh [hunk 3]
```diff
@@ -31,42 +35,67 @@ while getopts ":hcfm:" opt; do
...
 mkdir test_data_set_0
 mv *.pb test_data_set_0/
+
+
+if [ $CLONE = true ] ; then
+	if [ $CLEAN_DIR = true ] ; then
+		rm -rf $ONNX_MODELS_DIR
+	fi
+	git clone https://github.com/onnx/models.git $ONNX_MODELS_DIR
+fi
```
Comment (tsocha) [inline ngraph/python/tests/test_onnx/model_zoo_preprocess.sh:91]: it was added by mistake, wasn't it?
```suggestion
```

**Review by tsocha — CHANGES_REQUESTED** [review]

File: ngraph/python/tests/test_onnx/model_zoo_preprocess.sh [hunk 4]
```diff
@@ -31,42 +35,67 @@ while getopts ":hcfm:" opt; do
 		f )
 			CLEAN_DIR=true
 			;;
+        e )
+		    ENABLE_MSFT=true
+		    ;;
```
Comment (tsocha) [inline ngraph/python/tests/test_onnx/model_zoo_preprocess.sh:40]:
```suggestion
		f )
			CLEAN_DIR=true
			;;
		e )
			ENABLE_MSFT=true
			;;
```

**Review by tsocha — APPROVED** [review]

**Review by mbencer — APPROVED** [review]

**Pull request merged**